lms/resources/_js_config/__init__.py

```diff
@@ -667,13 +680,17 @@ def _configure_groups(self, course, assignment):
                 },
             }
 
-    def _get_lti_launch_debug_values(self):
+    def _get_lti_launch_debug_values(
+        self, course: Course, assignment: Assignment | None
```

Comment:
Setting `assignment` as `Assignment | None` because of this https://github.com/hypothesis/lms/pull/6263/files#r1598218951

I'll change that if I manage to figure that out.